.storybook/main.ts

```diff
@@ -2,21 +2,21 @@ import type { StorybookConfig } from "@storybook/react-vite";
 
 const config: StorybookConfig = {
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
+
   addons: [
     "@storybook/addon-onboarding",
     "@storybook/addon-links",
-    "@storybook/addon-essentials",
     "@chromatic-com/storybook",
-    "@storybook/addon-interactions",
     "@storybook/addon-themes",
     'storybook-dark-mode',
+    "@storybook/addon-docs",
+    '@vueless/storybook-dark-mode'
```

Comment:
Is `@vueless/storybook-dark-mdoe` used by storybook?

Reply:
Good spot @amorey 

`storybook-dark-mode` is an addon used to add dark-mode in storybook. But it seems the project is a bit outdated and doesn't support Storybook v9. 

Here is the issue https://github.com/hipstersmoothie/storybook-dark-mode/issues/295 with all details. One of recommendations was to switch to `@vueless/storybook-dark-mode`. I did this, but forgot to remove `storybook-dark-mode` from `main.ts`.